### ISSUE TITLE
use `bash` instead of `sh` for code 

### ISSUE THREAD

**Issue by manmeetkaur** [astronomer/docs#2274]
**Affected Document:** CLI docs

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [ ] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [x] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):** use `bash` instead of `sh`

**Additional Notes:** Reference: https://docs.astronomer.io/software/third-party-ingress-controllers#optional-share-a-certificate-between-astronomer-and-your-ingress-controller where comments and other keywords are colour-coded compared to this - https://docs.astronomer.io/astro/cli/astro-deployment-create#examples. This will improve readability.

**Comment by jwitz** [comment]
@manmeetkaur I added a note to our style guide. Is that enough to close this issue out?

When I did a find-replace, I found hundred of docs that would have been affected. I'd like to avoid updating all the docs at once so that our "docs freshness metric" still works in Sigma (if we update all the docs at once, even old docs will appear as having been recently updated, even though their content is the same).

![Screenshot 2023-08-15 at 2 22 14 PM](https://github.com/astronomer/docs/assets/74574233/a2ce727c-1c1a-449c-8ad7-cb467cd02f81)
